pkg/e2etest/e2e_test.go

```diff
@@ -141,20 +141,17 @@ func TestE2EMultipleIfaces(t *testing.T) {
 	for _, n := range []int{
 		2, 3, 5, 10, 21, 100,
 	} {
-		t.Run(fmt.Sprintf("%02d interfaces", n), func(t *testing.T) {
```

Comment:
Out of curiosity: What was wrong with the sub-test setup? :stuck_out_tongue:

Reply:
Sorry, this slipped in by accident. I will revert it!